CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Changed
 
+- Fixed problems caused by NAs for appendSpecificityMetrics
+- Fixed plotting 'groups' and 'genes' together, fixed warning when plotting shorter 'groups' than no. cells (embeddingPlot)
+
+## [1.1.3] - 2019-10-19
```

Comment:
Why did you add the new version? Increasing version requires more actions (see contribution guidelines) and happens only when changes are merged to master